android/app/src/main/java/org/fossasia/openevent/utils/DateUtils.java

```diff
@@ -166,4 +166,18 @@ public static String formatDay(@NonNull String isoString) throws ParseException
         return dates;
     }
 
+    public static String getRelativeTimeFromTimestamp(String timeStamp) {
+        String timeAgo=null;
+        try {
```

Comment:
Don't try and catch. throw the exception to caller to handle it so that he can do whatever he wants if it fails. This way increases the chance of NPE